store/cachekv/internal/memcache.go

```diff
@@ -0,0 +1,93 @@
+package internal
+
+import (
+	"bytes"
+	"errors"
+
+	"github.com/tidwall/btree"
+)
+
+var errKeyEmpty = errors.New("key cannot be empty")
+
+const (
+	// The approximate number of items and children per B-tree node. Tuned with benchmarks.
+	// copied from memdb.
+	bTreeDegree = 32
```

Comment:
What is the impact of misconfiguring this?

Reply:
I believe it only affect performance